spec/unit/document_spec.lua

```diff
@@ -80,6 +80,7 @@ describe("EPUB document module", function()
             "Noto Sans Arabic UI",
             "Noto Sans CJK SC",
             "Noto Sans Devanagari UI",
+            "Noto Sans Bengali UI",
```

Comment:
In here (and only in here), these need to be in alphabetical order, or the test fails.

Reply:
Okay, Great. Just pushed making this alphabetical. Hoping it to be resolved.